packages/orchestrator/Makefile

```
@@ -10,10 +10,10 @@ init:
 	go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.2
 
 .PHONY: generate
-generate:
+generate:generate-models
```

No `generate-models` exists 
```suggestion
generate:
```